pkgs/by-name/eb/ebpf-usb/package.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, bcc
+, makeWrapper
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ebpf-usb";
+  version = "unstable-2022-04-03";
+  format = "other";
```

```suggestion
  pyproject = false;
```
as noted in previous review comment